features/immigratie.feature

```diff
@@ -0,0 +1,67 @@
+#language: nl
+
+@proxy
+Functionaliteit: Immigratie
+
+  Rule: indicatieVestigingVanuitBuitenland wordt opgenomen met de waarde true wanneer datumVestigingInNederland een waarde heeft
+
+    Abstract Scenario: persoon heeft datumVestigingInNederland
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'immigratie' gegevens
+      | naam                                 | waarde            |
+      | land van waar ingeschreven (14.10)   | 9088              |
```

Comment:
Het is in principe niet nodig om dit veld op te nemen als gegeven. Voor het vullen van indicatieVestigingVanuitBuitenland wordt hier niet naar gekeken.